README.md

```diff
@@ -1,101 +1,96 @@
-# true[X]  Web Integrations
-Documentation and reference apps for integrating true[X]'s CTV web ad renderer
+# TruexAdRenderer Web Integration: Quick Guide
 
+This document describes the how to use Infillion's TrueX web ad renderer, a library that allows the presentation of
+interactive ads, intended for use in "Full Experience Playback" (FEP) scenarios such as video web sites or applications.
 
-## Setup
-This document describes the initial steps needed to make use of the `TruexAdRenderer` in an HTML5 web application intended for Smart TVs and game consoles, i.e. for the so-called "10 foot" experience.
+The library provides a class called the `TruexAdRenderer` (TAR). TAR is supports running within any web content, 
+specifically for desktop and mobile web content, as well as for the "Connected TV" (CTV) platforms, such as Smart TVs
+and game consoles, i.e. for the so-called "10 foot" experience.
 
-The true[X] ad renderer is available as an `npm` module. For the typical web app based around a `package.json` project file, one adds the true[X] dependency as follows:
+## Setup
```

Comment:
This section needs to include at least a mention that you can also just link the library, aka:

```javascript
 <script src="https://cdn.jsdelivr.net/npm/@truex/ctv-ad-renderer@1.9.0"></script>
```

Reply:
done